rero_ils/modules/holdings/jsonschemas/holdings/holding-v0.0.1.json

```diff
@@ -92,116 +113,167 @@
     },
     "holdings_type": {
       "type": "string",
-      "title": "Type of the holdings",
+      "title": "Holding Type",
       "enum": [
         "standard",
         "electronic",
         "serial"
       ],
-      "default": "standard"
+      "default": "serial",
+      "form": {
+        "hideExpression": "true"
+      }
     },
     "patterns": {
-      "title": "Captions for the patterns enumeration and chronology.",
+      "form": {
+        "expressionProperties": {
+          "templateOptions.required": "true"
+        }
+      },
+      "title": "Serial Configuration",
       "type": "object",
+      "propertiesOrder": [
+        "template",
+        "values"
+      ],
       "required": [
         "values",
         "template"
       ],
       "additionalProperties": false,
       "properties": {
         "template": {
-          "title": "Template for the display of the holdings issues.",
+          "title": "Issue Display Template",
+          "description": "This template define how a prediction is displayed. i.e. year 2020; vol.1, iss. 4. It should contains the variable field between {{}} with the corresponding pattern and level name: i.e. {{chronology.year}}; vol. {{enumeration.volume}}, iss. {{enumeration.issue}}.",
```

Comment:
replace `This template define how a prediction is displayed` by `This template define how an issue is displayed` ?